src/app/combo/combo.sample.html

```diff
@@ -106,6 +106,43 @@ <h1 class="combo-sample__title">Remote Binding</h1>
         </igx-simple-combo>
     </section>
 
+    <section class="combo-sample__section">
+        <h1 class="combo-sample__title">Open simple combo on clear by handling the selectionChanging event</h1>
+        <!-- Simple combo with selectionChanging event -->
```

Comment:
The title of the sample already explains what it's meant to represent, so no need for these comments imo.